spec/models/ingredient_spec.rb

```diff
@@ -27,4 +27,14 @@
       expect { ingredient.save! }.to raise_error(ActiveRecord::RecordInvalid)
     end
   end
+
+  context 'with many ingredients' do
+    let!(:ingredients_good) { create_list(:ingredient, 5, inventory: 20, minimum_quantity: 10) }
+    let!(:ingredients_bad) { create_list(:ingredient, 5, inventory: 20, minimum_quantity: 30) }
+    let!(:ingredients_nil) { create_list(:ingredient, 5, inventory: 20, minimum_quantity: nil) }
+
+    it 'scope get the correct amount of alerted ingredients' do
+      expect(Ingredient.below_minimum.count).to eq(ingredients_bad.count)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] RSpec/DescribedClass: Use described_class instead of Ingredient.